docs/development/build-instructions-gn.md

```diff
@@ -72,6 +72,32 @@ $ gclient sync --with_branch_heads --with_tags
 # This will take a while, go get a coffee.
 ```
 
+#### A note on pulling/pushing
+
+If you intend to `git pull` or `git push` from the official `electron`
+repository in the future, you now need to update the respective folder's
+origin URLs.
+
+```sh
+$ cd src/electron
+$ git remote remove origin
+$ git remote add origin https://github.com/electron/electron
```

Comment:
Maybe `git remote set-url origin` instead of `remote remove` and `remote add`?

Reply:
`git remote set-url origin` only sets both fetch and push if they are both the same, if they are not the same it only sets `fetch` and leaves `push` alone.